cgmes/cgmes-completion/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2017, RTE (http://www.rte-france.com)
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+-->
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <!-- <pattern>%-5p %d{HH:mm:ss.SSS} %-20C{1} | %m%n</pattern> -->
+            <pattern>%-5p %-20C{1} | %m%n</pattern>
+        </encoder>
+    </appender>
+    <root level="error">
+        <appender-ref ref="STDOUT"/>
+    </root>
+    <logger name="com.bigdata" level="error"/>
+    <logger name="com.powsybl.triplestore.TripleStore" level="error"/>
+    <logger name="com.powsybl.cgmes.triplestore.CgmesModelTripleStore" level="error"/>
+    <logger name="com.powsybl.cgmes.conversion.Conversion" level="error"/>
+    <logger name="com.powsybl.cgmes.completion" level="info"/>
```

Comment:
Why is the log level info here?

Reply:
For the unit tests, to have easy access to the relevant info about the run (location of the output file, list of missing voltage levels in the input) ...

Reply:
log level updated to `error`